text/src/autogluon/text/automm/configs/environment/default.yaml

```diff
@@ -0,0 +1,12 @@
+env:
+  num_gpus: -1
+  num_nodes: 1
+  batch_size: 128  # this is a desired batch size; pl trainer will accumulate gradients when per step batch is smaller.
+  per_gpu_batch_size: 16  # you can define this according to your GPU memory
+  precision: 16  # training precision
+  num_workers: 2  # pytorch training dataloader workers
+  num_workers_evaluation: 2  # pytorch prediction/test dataloader workers
+  fast_dev_run: False
+  deterministic: False
```

Comment:
When it is True, it will turn on `torch.cudnn.deterministic` that will force to use some deterministic kernels and throw error if there are only non-deterministic implementations (doc: https://pytorch.org/docs/stable/notes/randomness.html#avoiding-nondeterministic-algorithms). However, there are certain operations that are really difficult to be made deterministic. One example is the backward operation of affineTransform, which require the usage of "atomicAdd()". Thus, we may not turn on this flag.

Reply:
This does not mean that our method won't mostly be deterministic with the fixed random seed.